Main_Data_Classes_6.rb

```diff
@@ -0,0 +1,8 @@
+# 6. There is an array of integers (ary). Puts the index of the last element where ary[0]<ary[i]<ary[-1]
```

Comment:
This expression `ary[0]<ary[i]<ary[-1]` means that you need to compare values of the array elements